### PR TITLE
PP-1739 Open Sourcing

### PR DESCRIPTION
**Pull request by markyjones**
## WHAT
This was already a public repository but didn't have a licence or other supporting information. Unlike the others the readme's API spec is fairly small so I am not moving this out into a docs/ folder yet.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Crown Copyright (Government Digital Service)
```
Comment (alexbishop1) [inline LICENSE]: Is the year right? Earliest commit was on May 23, 2016.

Comment (markyjones) [inline LICENSE]: Clearly not :-) Will correct.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Crown Copyright (Government Digital Service)
```
Comment (alexbishop1) [inline LICENSE]: Of course, we’ve also made changes to it in 2017 as well!

Looked around and found http://softwareengineering.stackexchange.com/a/210491, which suggests — based on guidance from the FSF — that we should make it say “Copyright (c) 2016, 2017 Crown Copyright (Government Digital Service)” (and update it for every year in which we make copyrightable changes).

Comment (markyjones) [inline LICENSE]: Not sure I want to begin playing that game.

Comment (alexbishop1) [inline LICENSE]: Fair enough! I won’t object to leaving it as just the year of the earliest code.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+This guide covers the basics of how to contribute to the GOV.UK Pay project.
+
+## Pull requests
+The team's pull request checklist can be found [here](https://github.com/alphagov/pay-team-manual/blob/master/docs/development-processes/pull-request-checklist.md)
+
+## Contributions from beyond the team
+If you have an idea to share or a feature to request to raise please contact the GOV.UK Pay team govuk-pay-support@digital.cabinet-office.gov.uk. 
+
+If this is a security issue please do not submit a pull request or raise a Github issue, instead, please read the disclosure process [here](/README.md#responsible-disclosure).
```
Comment (alexbishop1) [inline CONTRIBUTING.md]: Change this to “GitHub” and I’ll approve!

Comment (markyjones) [inline CONTRIBUTING.md]: Done, cheers

**Pull request merged**